tools/docker/tests/test_flows.sh

```diff
@@ -127,8 +127,15 @@ test_client_steering_mandate() {
 }
 
 test_client_steering_policy() {
-    #TODO: Implement
-    return 1
+    status "test client steering policy"
+    eval send_bml_command '"bml_wfa_ca_controller \"DEV_SEND_1905,DestALid,$mac_agent1,MessageTypeValue,0x8003,tlv_type,0x89,tlv_length\
+,0x000C,tlv_value,{0x00 0x00 0x01 {0x112233445566 0x01 0xFF 0x14}}\""' $redirect
+    sleep 1
+    dbg "Confirming client steering policy has been received on agent"
+    docker exec -it repeater1 sh -c 'grep -i -q "MULTI_AP_POLICY_CONFIG_REQUEST_MESSAGE" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
```

Comment:
As mentioned in #280 - we should parse the MID here so we can match for it in the gateway log.